azure-pipelines.yml

```diff
@@ -0,0 +1,106 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+
+pr: none
+
+variables:
+  - group: 'Prod Management Resources'
+  - name: buildConfiguration
+    value: 'release'
+
+resources:
+  repositories:
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/heads/master
+    endpoint: "GitHub (SFA)"
+
+jobs:
+- job: 'CodeBuild'
+  pool:
+    name: 'DAS - Continuous Integration'
+    demands: Agent.OS -equals Linux
```

Comment:
```suggestion
```
The standard chosen is let the .NET Core tasks run on Windows/Linux. And work to be done so Linux would be picked if available as it's quicker.